_data/compose-cli/docker_compose_exec.yaml

```diff
@@ -35,17 +37,30 @@ options:
   description: |
     index of the container if there are multiple instances of a service [default: 1].
   deprecated: false
+  hidden: false
+  experimental: false
+  experimentalcli: false
+  kubernetes: false
+  swarm: false
+- option: interactive
+  shorthand: i
+  value_type: bool
+  default_value: "true"
+  description: Keep STDIN open even if not attached.
+  deprecated: false
+  hidden: true
   experimental: false
   experimentalcli: false
   kubernetes: false
   swarm: false
 - option: no-TTY
   shorthand: T
   value_type: bool
-  default_value: "false"
+  default_value: "true"
   description: |
     Disable pseudo-TTY allocation. By default `docker compose exec` allocates a TTY.
```

Comment:
Is the TTY now auto-detected on `compose exec` as well? (like on `compose run`?) or is it hard-coded to "enabled"?

Reply:
It's autodetected 
https://github.com/docker/compose/blob/4cebef1bf11c9ded9e2bdd405071f9340ed79e59/cmd/compose/exec.go#L72